### PR TITLE
fi-1408-update-links

### PR DESCRIPTION
**Pull request by Jack-Fraser16**

### CODE REVIEW

File: lib/onc_certification_g10_test_kit/bulk_data_authorization.rb [hunk 1]
```diff
@@ -6,8 +6,8 @@ class BulkDataAuthorization < Inferno::TestGroup
     short_description 'Demonstrate SMART Backend Services Authorization for Bulk Data.'
     description <<~DESCRIPTION
       Bulk Data servers are required to authorize clients using the
-      [Backend Service Authorization](http://hl7.org/fhir/uv/bulkdata/STU1/authorization/)
-      specification as defined in the [FHIR Bulk Data Access IG v1.0.0](http://hl7.org/fhir/uv/bulkdata/STU1/).
+      [Backend Service Authorization](https://hl7.org/fhir/uv/bulkdata/authorization/index.html)
+      specification as defined in the [FHIR Bulk Data Access IG v2.0.0](http://hl7.org/fhir/uv/bulkdata/).
```
Comment (Jammjammjamm) [inline lib/onc_certification_g10_test_kit/bulk_data_authorization.rb:10]: We need all of these links to point at bulk data v1.

File: lib/onc_certification_g10_test_kit/smart_standalone_patient_app_group.rb [hunk 2]
```diff
@@ -59,8 +59,8 @@ class SmartStandalonePatientAppGroup < Inferno::TestGroup
         # Background
 
         The [Standalone
-        Launch](http://hl7.org/fhir/smart-app-launch/#standalone-launch-sequence)
-        Sequence allows an app, like Inferno, to be launched independent of an
+        Launch Sequence](http://hl7.org/fhir/smart-app-launch/app-launch.html#launch-app-standalone-launch)
```
Comment (Jammjammjamm) [inline lib/onc_certification_g10_test_kit/smart_standalone_patient_app_group.rb:62]: Likewise, these all need to point to SMART App Launch v1. The links in the SMART App Launch test kit also need to be updated.

Comment (Jack-Fraser16) [reply lib/onc_certification_g10_test_kit/smart_standalone_patient_app_group.rb:62]: Okay, so would this be correct: http://hl7.org/fhir/smart-app-launch/1.0.0/index.html#standalone-launch-sequence

Comment (Jammjammjamm) [reply lib/onc_certification_g10_test_kit/smart_standalone_patient_app_group.rb:62]: Yes.

**Review by Jammjammjamm — APPROVED** [review]

**Pull request merged**